src/pages/Trends/connectedWordsEpic.spec.js

```diff
@@ -213,12 +213,12 @@ describe('Connect Trending Words', () => {
       ({ tags, ...insight }) => insight
     )
 
-    expect(result.payload.TrendToInsights).toEqual({
+    /* expect(result.payload.TrendToInsights).toEqual({
```

Comment:
@haritonasty @Partysun Does this test fails on your local machine?

Reply:
Only on my local machine :(

Reply:
If that's the case, then the test should be uncommented

Reply:
Ok